src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -12,7 +12,9 @@ internal enum ConversionKind : byte
         Identity,
         ImplicitNumeric,
         ImplicitEnumeration,
+        ImplicitTupleLiteral,
```

Comment:
You have a good point though.

What happens to
```  int x = 1; y = (byte, string)(x, null) ```

I think it should fail, but we can discuss if this is something worth enabling.